[2025-05-25T19:39:31.075+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Marketing_Data_Pipeline.validate_facts.validate_form_fills manual__2025-05-25T19:37:32.889912+00:00 [queued]>
[2025-05-25T19:39:31.369+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Marketing_Data_Pipeline.validate_facts.validate_form_fills manual__2025-05-25T19:37:32.889912+00:00 [queued]>
[2025-05-25T19:39:31.370+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-25T19:39:32.547+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): validate_facts.validate_form_fills> on 2025-05-25 19:37:32.889912+00:00
[2025-05-25T19:39:32.571+0000] {standard_task_runner.py:60} INFO - Started process 20663 to run task
[2025-05-25T19:39:32.589+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Marketing_Data_Pipeline', 'validate_facts.validate_form_fills', 'manual__2025-05-25T19:37:32.889912+00:00', '--job-id', '151', '--raw', '--subdir', 'DAGS_FOLDER/Marketing_Data_Pipeline.py', '--cfg-path', '/tmp/tmpbsm5vs4b']
[2025-05-25T19:39:32.620+0000] {standard_task_runner.py:88} INFO - Job 151: Subtask validate_facts.validate_form_fills
[2025-05-25T19:39:33.219+0000] {task_command.py:423} INFO - Running <TaskInstance: Marketing_Data_Pipeline.validate_facts.validate_form_fills manual__2025-05-25T19:37:32.889912+00:00 [running]> on host fe289fe396bd
[2025-05-25T19:39:34.045+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='yashbhawsar872@gmail.com' AIRFLOW_CTX_DAG_OWNER='Yash' AIRFLOW_CTX_DAG_ID='Marketing_Data_Pipeline' AIRFLOW_CTX_TASK_ID='validate_facts.validate_form_fills' AIRFLOW_CTX_EXECUTION_DATE='2025-05-25T19:37:32.889912+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-25T19:37:32.889912+00:00'
[2025-05-25T19:39:34.075+0000] {Marketing_Data_Pipeline.py:39} INFO - ▶️ Starting task: Validate Form Fills
[2025-05-25T19:39:34.221+0000] {data_validation.py:243} INFO - ⏩ Already validated: form_fills_2025-05-25.parquet
[2025-05-25T19:39:34.231+0000] {utils.py:55} INFO - task_metrics
[2025-05-25T19:39:34.237+0000] {Marketing_Data_Pipeline.py:41} INFO - ✅ Completed task: Validate Form Fills
[2025-05-25T19:39:34.242+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-25T19:39:34.299+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Marketing_Data_Pipeline, task_id=validate_facts.validate_form_fills, execution_date=20250525T193732, start_date=20250525T193931, end_date=20250525T193934
[2025-05-25T19:39:34.449+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-25T19:39:34.665+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
